### ISSUE TITLE
[sources.src] url=nosource for metapackages

### ISSUE THREAD

**Issue by christophprokop** [science-computing/butido#336]
At the moment we use dummy source for meta packages which shouldn't be necessary at all.
for example pkg.toml could contain a section like this:

```
[sources.src]
url=nosource
# hash.hash should not be needed then as well
```

**Comment by primeos-work** [comment]
@christophprokop I'd propose that we try to make specifying a source optional so that `sources.src` can be omitted entirely (seems like the cleanest solution) - would that be fine as well?

**Comment by christophprokop** [comment]
@primeos-work yes, that would help to clean up all the configurations that use dummy files at the moment
